Misc/NEWS.d/next/Library/2022-01-07-13-27-53.bpo-46246.CTLx32.rst

```diff
@@ -0,0 +1,2 @@
+Add missing ``__slots__`` to ``importlib.metadata.DeprecatedList``. Patch by
```

Comment:
```suggestion
Add missing ``__slots__`` to :class:`importlib.metadata.DeprecatedList`. Patch by
```

Reply:
I decided not to include a link because AFAIK ``DeprecatedList`` isn't documented anywhere (the link will not work).

Do we still want to add `:class:` even if the target doesn't exist?